node_cli/core/resources.py

```diff
@@ -186,3 +205,8 @@ def get_allocation_option_name(schain):
 def get_disk_path():
     with open(DISK_MOUNTPOINT_FILEPATH) as f:
         return f.read().strip()
+
+
+def init_shared_space_volume(env_type):
+    size = get_shared_space_size(env_type)
```

Comment:
If size is None function will fail with strange error. Better raise meaningful exception.